.github/workflows/staging-cd.yml

```
@@ -12,14 +12,12 @@ jobs:
       - name: Install libvips
         run: sudo apt-get install libvips
       - uses: actions/checkout@v2
-      - name: Setup Node 18
+      - name: Setup Node 20
         uses: actions/setup-node@v2
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)